e2e/test/iothub/service/DigitalTwinClientE2ETests.cs

```diff
@@ -47,13 +45,13 @@ public async Task DigitalTwinWithOnlyRootComponentOperationsAsync()
                 await deviceClient.OpenAsync().ConfigureAwait(false);
 
                 // Perform operations on the digital twin.
-                using var digitalTwinClient = DigitalTwinClient.CreateFromConnectionString(s_connectionString);
+                using var serviceClient = new IotHubServiceClient(s_connectionString);
 
                 // Retrieve the digital twin.
-                HttpOperationResponse<ThermostatTwin, DigitalTwinGetHeaders> response =
-                    await digitalTwinClient.GetDigitalTwinAsync<ThermostatTwin>(deviceId).ConfigureAwait(false);
-                ThermostatTwin twin = response.Body;
-                twin.Metadata.ModelId.Should().Be(ThermostatModelId);
+                GetDigitalTwinResponse<ThermostatTwin> response =
+                    await serviceClient.DigitalTwins.GetAsync<ThermostatTwin>(deviceId).ConfigureAwait(false);
```

Comment:
Other cases like this too:
```suggestion
                GetDigitalTwinResponse<ThermostatTwin> response = await serviceClient.DigitalTwins
                    .GetAsync<ThermostatTwin>(deviceId)
                    .ConfigureAwait(false);
```